src/screens/NationsScreen/NationsListScreen/index.js

```diff
@@ -58,6 +58,7 @@ class NationsListScreen extends Component {
             return (<NationListItem text={nation.nationName}
                                     onPress={this.props.onSelectItem}
                                     status={i18n.t(`enums.nation.status.${resolveStatus(nation)}`)}
+                                    statusColor={resolveListStyle(nation)}
```

Comment:
If you return a color name, please change the method name, so it doesn't look as if it returns style object

Reply:
Changed the name to `statusColor`